.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: python
 sudo: false
-python:
-    - 2.6
-    - 2.7
-    - 3.3
-    - 3.4
-    - pypy
-    - pypy3
+
+env:
+  - TOXENV=py26
+  - TOXENV=py27
+  - TOXENV=py32
+  - TOXENV=py33
+  - TOXENV=py34
+  - TOXENV=pypy
+  - TOXENV=pypy3
+
 install:
-    - pip install .
+  - travis_retry pip install tox
+
 script:
-    - python setup.py test -q
+  - travis_retry tox
```

Comment:
If this does what I think it does, then please no.

Retrying the entire test suite three times because a test fails makes the job four times slower for no good reason -- unless you're trying to work around unfixable nondeterministically failing flakey tests?